wagtail-{{ cookiecutter.project_name_kebab }}/README.md

```diff
@@ -34,3 +34,52 @@
 
 - `pip install {{ cookiecutter.project_name_kebab }}`
 - ...
+
+
+## Contributing
+
+To make changes to this project, first clone this repository:
+
+```sh
+git clone git@https://github.com/wagtail/{{ cookiecutter.project_name_kebab }}.git
+cd {{ cookiecutter.project_name_kebab }}
+```
```

Comment:
```
git clone git@https://github.com/{{ cookiecutter.github_username }}/wagtail-{{ cookiecutter.project_name_kebab }}.git
cd wagtail-{{ cookiecutter.project_name_kebab }}
```

Reply:
Side note: I'd quite like to make some variables in the cookiecutter setup that include the `wagtail-` prefix but I'd like to try and get some of this open stuff picked off first before I shave that specific yak